cli/args.go

```diff
@@ -85,6 +85,8 @@ func parseTerragruntOptionsFromArgs(args []string, writer, errWriter io.Writer)
 
 	ignoreExternalDependencies := parseBooleanArg(args, OPT_TERRAGRUNT_IGNORE_EXTERNAL_DEPENDENCIES, false)
 
+	IncludeExternalDependencies := parseBooleanArg(args, OPT_TERRAGRUNT_INCLUDE_EXTERNAL_DEPENDENCIES, false)
```

Comment:
Should be lower case by Go's naming conventions. Upper case is only used for public fields (e.g., on a struct, as the field name on `terragrunt.Options`).

Reply:
Fixed that, sorry!

Reply:
Thx!